# HG changeset patch
# User Thomas Weber <thomas.weber.mail@gmail.com>
# Date 1224009296 14400
#      Tue Oct 14 14:34:56 2008 -0400
# Node ID 518789a0539db871e29fb8e7aee7b4974602b977
# Parent  263261b2124fbc0f55ecf4b8bfcd347035bb9a37
FAQ entry about split development packages in distributions

diff --git a/doc/faq/Octave-FAQ.texi b/doc/faq/Octave-FAQ.texi
--- a/doc/faq/Octave-FAQ.texi
+++ b/doc/faq/Octave-FAQ.texi
@@ -644,16 +644,34 @@ line 26317: invalid command}.
 
 There is a known bug in gnuplot 4.2 that can cause an off by one error
 while piping data to gnuplot. The relevant gnuplot bug report can be
 found at @url{http://sourceforge.net/tracker/index.php?func=detail&aid=1716556&group_id=2055&atid=102055}
 
 If you have obtained your copy of Octave from a distribution please file
 a bug report requesting that the fix reported in the above bug report be
 included.
+
+@item
+I cannot install a package. Octave complains about a missing @code{mkoctfile}.
+
+Most distributions split Octave into several packages. The script
+@code{mkoctfile} is then part of a separate package:
+@itemize @minus
+@item
+Debian/Ubuntu
+
+@code{aptitude install octave3.0-headers}
+
+@item
+Fedora
+
+@code{yum install octave-devel}
+
+@end itemize
 @end itemize
 
 @node How do I ...?
 @chapter  How do I ...?
 
 @menu
 * How do I set the number of displayed decimals?::
 @end menu
